### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by ryan1604**

### CODE REVIEW

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -8,24 +8,24 @@
...
     * [List all](#list-all-list)
     * [List income](#list-income-list-income)
     * [List expense](#list-expense-list-expense)
     * [List recurring](#list-recurring-list-recurring)
+  * [Delete cashflow](#delete-cashflow-delete)
+    * [Delete income](#delete-income-delete-income)
+    * [Delete expense](#delete-expense-delete-expense)
+    * [Delete recurring cashflow](#delete-recurring-delete-recurring)
+  * [Find cashflow](#find-cashflow-find)
+  * [View Balance](#viewing-balance-balance)
```
Comment (wwweert123) [inline docs/UserGuide.md:21]: Nice 👍

File: docs/UserGuide.md [hunk 2]
```diff
@@ -77,6 +77,8 @@ you a one-stop interface to access a plethora of features to manage your finance
 - Minimum amount for each cashflow and total balance that the program can hold is -999,999,999,999.99
 - Total Balance, Income balance, and Expense balance are different entities where the latter two do not have the same limitations.
 
+**Important:** Data is automatically loaded on start up and saved when exited. You must exit the program using the `exit` command in order to save your changes.
```
Comment (wwweert123) [inline docs/UserGuide.md:80]: Good clarity 👍

File: src/main/java/seedu/financialplanner/commands/BudgetCommand.java [hunk 3]
```diff
@@ -99,10 +99,10 @@ private void validateCommandFormat(RawCommand rawCommand) throws FinancialPlanne
 
         if (command.equals("set") && Budget.hasBudget()) {
             logger.log(Level.WARNING, "Invalid command: Trying to set existing budget");
-            throw new FinancialPlannerException("There is an existing budget, did you mean update?");
+            throw new FinancialPlannerException("There is an existing budget, try budget update instead.");
```
Comment (wwweert123) [inline src/main/java/seedu/financialplanner/commands/BudgetCommand.java:102]: 👍

**Review by wwweert123 — APPROVED** [review]
LGTM

**Pull request merged**